deploy/signup-uitestrig/install.sh

```
@@ -47,42 +61,120 @@ function installing_uitestrig() {
+ while [[ ! " ${valid_inputs[@]} " =~ " ${eSignetDeployed} " ]]; do
+     read -p "Is the esignet service deployed? (yes/no): " eSignetDeployed
+     eSignetDeployed=${eSignetDeployed,,}  # Convert input to lowercase
+ done
+
+ if [[ $eSignetDeployed == "yes" ]]; then
+     echo "esignet service is deployed. Proceeding with installation..."
+ else
+     echo "esignet service is not deployed. hence will be skipping esignet related test-cases..."
+ fi
+  read -p "Is values.yaml for uitestrig chart set correctly as part of pre-requisites? (Y/n) : " yn;
+  if [[ $yn = "Y" ]] || [[ $yn = "y" ]] ; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)